content/installation/setup/auth/ConfigureAD2.md

```diff
@@ -1,136 +1,122 @@
+* **Username:** The username of a user who can bind to the directory to perform search functionality. (The full DN of the user account that the application should use when connecting to the LDAP server.) The default is *cn=Directory Manager*. See the section to **Create a service user** for more information.
 
-## Configure Groups
-Contrast doesn't perform Data Access Control using the integrated Active Directory Servers. In other words, roles and access to data within the application are handled by the application, and user roles are set by the Organization Administrators. However, there is an Access Control check when logging in or creating new users to ensure that the provided user belongs to the correct group in the Active Directory.
+* **Password:** The password of the user account that the application should use when connecting to the LDAP server. The default is "N/A".
 
-**Contrast User Group:**: This group allows users to be added to an organization and log in to the standard user interface. Most users should be part of this group. 
+After configuring your connection details and bind credentials, select the button to **Test Connection** to ensure connectivity to the server. Once connectivity is verified, click **Next**. 
 
-**Contrast SuperAdmin Group:** This group allows users to log in to the Super Administrator interface. This interface should only be accessible to Application and System Administrators.
+### Step 3: Configure Groups
 
-**A user can't belong to both the User and Admin groups. If you add them to both groups in your AD instance, Contrast won't be able to find them during configuration.** 
+Contrast doesn't perform Data Access Control using the integrated AD servers. In other words, roles and access to data within the application are handled by the application, and user roles are set by the Organization Administrators. However, there is an Access Control check when logging in or creating new users to ensure that the provided user belongs to the correct group in AD. 
 
 <a href="assets/images/KB4-c09_2.png" rel="lightbox" title="Configuring Groups"><img class="thumbnail" src="assets/images/KB4-c09_2.png"/></a>
 
-> **Note:** You can use the **Query for Groups** button to enable a live search of existing groups as you begin to type within the input fields.
+* **Contrast User Group:**: This group allows users to be added to an organization and log in to the standard user interface. Most users should be part of this group. 
+
+* **Contrast SuperAdmin Group:** This group allows users to log in to the Super Administrator interface. This interface should only be accessible to Application and System Administrators.
 
-## Advanced Settings
+> **Note:** A user can't belong to both the User and Admin groups. **If you add them to both groups in your AD instance, Contrast won't be able to find them during configuration.** 
```

Comment:
If you add it in both groups, it will continue to work, only that the user will act as part of the Superadmin Group